example/postgres/multithread/main.go

```diff
@@ -0,0 +1,43 @@
+package main
```

Comment:
Let's call this example something different? Go engine is already multithreaded. Maybe you are looking for `multi-tenancy`?